common/lib/platform.sh

```diff
@@ -49,7 +51,45 @@ passthrough_envars() {
     done
 
     echo "Warning: Will pass env. vars. matching the following regex:
-$passthrough_env_re" > /dev/stderr
+$passthrough_env_re" >> /dev/stderr
 
     compgen -A variable | grep -Ev "$SECRET_ENV_RE" | grep -E  "$passthrough_env_re"
 }
+
+# On more occasions than we'd like, it's necessary to put temporary
+# platform-specific workarounds in place.  To help ensure they're
+# actually be temporary, it's useful to place a time-limit on them.
+# This function accepts two arguments:
+# - A (required) future, date/time of the form YYYYMMDD (UTC based).
+# - An (optional) message string to present after the 1st argument date
+timebomb() {
+    local expire="$1"
+
+    if ! expr "$expire" : '[0-9]\{8\}$' >> /dev/null; then
+      echo "timebomb: '$expire' must be UTC-based and of the form YYYYMMDD"
+      exit 1
+    fi
+
+    if [[ $(date -u +%Y%m%d) -lt $(date -u -d "$expire" +%Y%m%d) ]]; then
+        return
+    fi
+
+    declare -a frame
+    read -a frame < <(caller)
+
+    cat << EOF >> /dev/stderr
```

Comment:
`>&2` please. We've been burned before by `>/dev/stderr`. Let's please never use it again, it is evil and nonstandard.

Reply:
Okay.  And :sob: https://github.com/containers/automation/pull/175